crates/biome_js_analyze/tests/specs/nursery/useGuardForIn/invalid.js.snap

```diff
@@ -0,0 +1,110 @@
+---
+source: crates/biome_js_analyze/tests/spec_tests.rs
+expression: invalid.js
+---
+# Input
+```jsx
+for (var x in o) { if (x) { f(); continue; } g(); }
+for (var x in o) { if (x) { continue; f(); } g(); }
+for (var x in o) { if (x) { f(); } g(); }
+for (var x in o) { if (x) f(); g(); }
+for (var x in o) { foo() }
+for (var x in o) foo();
+```
+
+# Diagnostics
+```
+invalid.js:1:1 lint/nursery/useGuardForIn ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+
+  ! Require `for-in` loops to include an `if` statement.
```

Comment:
The message should be improved. In this case, there's a `if` statement, so I was a user, I wouldn't know what to do.  I would suggest an alternative, but at the moment the documentation is poor, so I don't understand why there's a diagnostic for this case.

Reply:
sorry for that, i will add more detailed message for it.

Reply:
add a message here, please review again for you free.